custom_components/myskoda/coordinator.py

```diff
@@ -261,5 +269,19 @@ def _debounce(
     ) -> RefreshFunction:
         return MySkodaDebouncer(self.hass, func, immediate).async_call
 
+    async def _update_positions(self) -> None:
+        _LOGGER.debug("Updating positions for %s", self.vin)
+        try:
+            await asyncio.sleep(60)  # GPS is not updated immediately, wait 60 seconds
```

Comment:
nit: With a debounce of 30s (default) and a sleep of 60s we possibly have a small list of outstanding events. Not terribly important now, but at some poing we may want to debounce this at 60.